react/index.tsx

```diff
@@ -84,6 +106,7 @@ const AddressChallenge: StorefrontFunctionComponent<WrappedComponentProps &
       })
         .catch(() => null)
         .then(() => {
+          updateRegionID()
```

Comment:
To continue from the above comment, you would then do this:
```suggestion
          updateRegionID(addressFields)
```